math.js

```diff
@@ -28,3 +28,11 @@ function testAdd() {
 function subtract(a, b) {
   return a - b;
 }
+
+function power(a, b) {
+  let product = a;
```

Comment:
- what will happen if a and b are zero? 
- please add some pre condition checks here.

Reply:
resolved